### ISSUE TITLE
action render_values UnicodeEncodeError

### ISSUE THREAD

**Issue by maxim-s-barabash** [StackStorm/st2#3421]
Error occurred when executing an action. if the variable value of the variable can not be converted to a string.

```
---
vars:
   unicode_var: 'ыыыы'
chain:
    -
        name: "c1"
        ref: "core.local"
        parameters:
            cmd: "echo {{ unicode_var }}"
```

```
Traceback (most recent call last):
  File "/opt/stackstorm/runners/action_chain_runner/action_chain_runner.py", line 295, in run
    self.chain_holder.init_vars(action_parameters)
  File "/opt/stackstorm/runners/action_chain_runner/action_chain_runner.py", line 78, in init_vars
    action_parameters=action_parameters)
  File "/opt/stackstorm/runners/action_chain_runner/action_chain_runner.py", line 203, in _get_rendered_vars
    return jinja_utils.render_values(mapping=vars, context=context)
  File "/usr/local/lib/python2.7/dist-packages/st2common/util/jinja.py", line 137, in render_values
    v = str(v)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-3: ordinal not in range(128)
```

**Comment by Kami** [comment]
Thanks for reporting this, will look into a fix shortly.

**Comment by Kami** [comment]
#3423 should fix this.

**Comment by Kami** [comment]
#3423 has been merged, fix will be included in v2.3.0.

Thanks again for reporting this.

**Comment by descrepes** [comment]
Hi,

I just upgraded to v2.2 and the same issue appear on Mistral workflow.
Here is the mistral server logs:

> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server [-] Exception during message handling
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server Traceback (most recent call last):
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 155, in _process_incoming
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     res = self.dispatcher.dispatch(message)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/oslo_messaging/rpc/dispatcher.py", line 222, in dispatch
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     return self._do_dispatch(endpoint, method, ctxt, args)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/oslo_messaging/rpc/dispatcher.py", line 192, in _do_dispatch
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     result = func(ctxt, **new_args)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/mistral/engine/engine_server.py", line 151, in on_action_complete
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     (rpc_ctx, action_ex_id, result.cut_repr())
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/mistral/workflow/utils.py", line 44, in cut_repr
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     utils.cut(self.data), utils.cut(self.error), str(self.cancel)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/mistral/utils/\_\_init\_\_.py", line 283, in cut
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     return cut_dict(data, length=length)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server   File "/opt/stackstorm/mistral/local/lib/python2.7/site-packages/mistral/utils/\_\_init\_\_.py", line 197, in cut_dict
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server     v = str(value)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 1047: ordinal not in range(128)
> 2017-06-11 03:44:08.389 52240 ERROR oslo_messaging.rpc.server

I never seen this kind of unicode error on v2.1.

I fix the code as in #3423 but applied to the file mistral/utils/\_\_init\_\_.py and everything seems ok so far :)

Regards
